data/data/powervs/variables-powervs.tf

```diff
@@ -46,6 +46,45 @@ variable "cloud_instance_id" {
   default     = ""
 }
 
+################################################################
+# Configure storage
+################################################################
+variable "cos_resource_group_name" {
+  type = string
+  description = "The name of the COS resource group that holds your resource"
+  default = ""
+}
+
+variable "cos_instance_name" {
+    type        = string
+    description = "The name of the COS instance that will hold your bucket"
+    default     = ""
+}
+
+variable "cos_instance_plan" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "standard"
+}
+
+variable "cos_instance_location" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "global"
+}
+
+variable "cos_storage_class" {
+    type        = string
+    description = "The cloud instance ID of your account"
+    default     = "standard"
+}
+
+variable "cos_bucket_location" {
```

Comment:
Can we set some default value to this value to avoid too many params or set to some region based on the region of the powervs instance?

Reply:
We should discuss this further, I don't think it's a given that your storage will be in the same location as your compute. For now, I'm setting to us-east

Reply:
i think a default is fine, but we should try and override that using the region specified in the install-config

Reply:
you are right, we need to discuss further on this because today I did some testing had difficulty accessing the us-south pre-signed URLs from my laptop(India),